src/components/views/context_menus/MessageContextMenu.js

```diff
@@ -130,6 +130,8 @@ export default class MessageContextMenu extends React.Component {
             roomId: ev.getRoomId(),
             eventId: ev.getId(),
             content: ev.event,
+            isEncrypted: this.props.mxEvent.getType() !== this.props.mxEvent.getWireType(),
```

Comment:
Hmm, could we instead use `ev.isEncrypted()` here, or is there some reason it fails? (I realise you're likely just moving the pre-existing check...) If we cannot use `isEncrypted`, we should probably still at least use `ev` instead of `this.props.mxEvent` I think...?

Reply:
Initially I moved the pre-existing check (I'm not sure about the initial motivation)
I did not see any difference in my tests, so isEncrypted seems to be fine?